### PR TITLE
doc: update snippets with first-period entry

### PR DESCRIPTION
**Pull request by malhussan**

### CODE REVIEW

File: docs/meshstack.billing-configuration.md [hunk 1]
```diff
@@ -362,8 +362,14 @@ let Statements =
         It is the offset of days after which chargeback statements are generated.
         This time should always be set to a higher value than the finalizeReportsAfterDays
         that are configured for kraken-worker, as only finalized reports are considered in chargeback statements.
+
+      first-period:
+        The chargeback statment period generation start date (UTC datetime format: yyyy-MM-ddTHH:mm:ssZ)
```
Comment (sttomm) [inline docs/meshstack.billing-configuration.md:367]:
```suggestion
       Chargeback statement periods will be created monthly starting from this date on (UTC datetime format: yyyy-MM-ddTHH:mm:ssZ)
```

File: docs/meshstack.billing-configuration.md [hunk 2]
```diff
@@ -377,13 +383,15 @@ let example
         , "paymentAmount"
         ]
       , period-offset-days = 10
+      , first-period = "2021-01-01T00:00:00Z"
```
Comment (sttomm) [inline docs/meshstack.billing-configuration.md:386]: d: You removed the config for VW and EnBW now completely? But at VW it must be set to "2020-12-01" and at enbw it must remain at "2020-01-01", because otherwise we will generate more periods of the past, which we don't want. Why did you remove the configs completely? Was there some confusion coming up due to our recent discussions around it?

**Review by sttomm — CHANGES_REQUESTED** [review]

File: docs/meshstack.billing-configuration.md (hunk 2 above)
Comment (malhussan) [reply docs/meshstack.billing-configuration.md:386]: Sorry for the confusion; this is the example part in the docs only. In the original PR https://github.com/meshcloud/meshfed-release/pull/840 VW is changed to 2020-12-01 and EnBW will remain using the default `application.yml` which is still on 2020-01-01.

File: docs/meshstack.billing-configuration.md (hunk 1 above)
Comment (malhussan) [reply docs/meshstack.billing-configuration.md]: Done

File: docs/meshstack.billing-configuration.md (hunk 2 above)
Comment (malhussan) [reply docs/meshstack.billing-configuration.md]: I updated the example to also be similar to `application.yml` values

**Review by sttomm — APPROVED** [review]
